setup.py

```diff
@@ -169,10 +169,7 @@ def package_files(prefix: str, directory: str, extension: str):
     'pycocotools>=2.0.4,<3',
 ]
 
-extra_deps['nlp'] = [
-    'datasets>=2.4,<3',
-    'transformers>=4.11,<5',
-]
+extra_deps['nlp'] = ['transformers>=4.11,<5', 'datasets>=2.4,<3']
```

Comment:
```suggestion
extra_deps['nlp'] = [
    'transformers>=4.11,<5',
    'datasets>=2.4,<3',
]
```

Reply:
nit: to be consistent with style elsewhere

Reply:
this was yapf

Reply:
Probably because there was no trailing comma